packages/oidf-client/package.json

```diff
@@ -1,5 +1,6 @@
 {
 	"name": "@sphereon/ssi-sdk.oidf-client",
+	"type": "module",
```

Comment:
hmm is this a change we should make?

Reply:
No idea, it's from John's branch SDK-44 which I needed for the new oidf-client. He must have added it for a reason

Reply:
so this PR is now merging in SDK-44? this is not what we should do then? this branch should then have been based on SDK-44?

Reply:
also setting type to module specifies the type of modules that the project will use. and this is the only one with this option. and from my experience is most likely never the solution to the problem

Reply:
Fixed, SDK-44 is merged also, then I synched up with develop